src/pages/installing/installing.md

```diff
@@ -137,7 +137,7 @@ To add the pull secret to individual ACD operand service accounts:
        --docker-username=<username> \
        --docker-password=<password> \
        --docker-email=<email_address> \
-       --namespace=<namespace>
+       --namespace=${acd_namespace}
```

Comment:
Do we want to add a note about setting the ${acd_namespace} variable first?  The 'Set Up Namespace' page has some details, but I'm thinking of the case where the reader might not read each page sequentially.

Reply:
I thought about that, but didn't think it was necessary on every page where it's used, especially for the installing piece. They'd be creating the namespace for a new install anyway so quite likely already visited the set up namespace page. We do also have some similar guidance on the migration pages so I felt we were covered. If you think it's necessary, go ahead and add it where you think it's needed for clarification.